workers/unity/Assets/Playground/Scripts/Cubes/ScoreCollision.cs

```diff
@@ -0,0 +1,40 @@
+using Generated.Playground;
+using Improbable.Gdk.Core;
+using Unity.Entities;
+using UnityEngine;
+
+public class ScoreCollision : MonoBehaviour
+{
+    private EntityManager entityManager;
+    private SpatialOSComponent component;
+
+    public void OnCollisionEnter(Collision col)
+    {
+        if (entityManager == null)
+        {
+            component = GetComponent<SpatialOSComponent>();
+            if (!component)
+            {
+                Debug.LogError("Could not get SpatialOSComponent.");
```

Comment:
Use the logger rather than Debug.Log

Reply:
Potentially problematic if we don't have access to a view. I'll keep it as is right now.